uniffi_macros/src/util.rs

```diff
@@ -212,6 +212,26 @@ mod kw {
 
 #[derive(Default)]
 pub(crate) struct CommonAttr {
+    /// Specifies the `UniFfiTag` used when implementing `FfiConverter`
+    ///   - When things are defined in the proc-macros, this is `None` which means create a blanket
+    ///     impl for all types.
```

Comment:
This is not entirely true, I think right now it's still allowed to specify a tag when using the derive. I can post a PR to remove that possibility though.

Reply:
Please do, I was wondering about that.